Semantic network database

AndriesSHP opened the pull request:
New version of Database uploaded to replace version in master.
Also Readme file improved.

AndriesSHP committed:
CreateDB and load test data

Initial module creating two kinds of tables,
Loading data and searching for data.

AndriesSHP committed:
Import sequence description added

AndriesSHP committed:
Read CSV and store in Database

The conversion of columns, based on the column IDs creates an error message. What is wrong?

Boudewijn26 commented on ImportInDB.py:
```diff
@@ -0,0 +1,81 @@
+    Read a file in a Gellish Expression Format.
+    Rearrange in expressions and Store expressions in a Gellish database.
+    """
+    def __init__(self, fname):
```
This method is quite long (~60 lines). It would be more readable if it was split up in several methods.

Commit:
Committer: Martijn Brouwer <martijnb@gmx.com>
Small fixes to Database.py and ImportInDB.py; added data files under Data and put them under version control

martijnzm committed:
Another typo in Database.py

AndriesSHP committed:
Double quotes at the end of line 5 removed

Unsure whether that caused the error message

AndriesSHP committed:
New version created

These files replace all current Python files and includes the proposed improvements.
The DatabaseAPI was used to create the SQlite3 FormalEnglishDB database from a conversion of the Taxonomic Dictionary of Formal English (core).csv file that is stored in subdirectory Data.

AndriesSHP committed:
Delete Database.py; replaced by DatabaseAPI.py

New module includes import from CSV files.

AndriesSHP committed:
Replaced by InitializeFormalLanguage.py

Delete FormalLanguage.py
It was only initialization. The formal language definition consists of an import of the whole taxonomic dictionary-ontology.

AndriesSHP committed:
Delete ImportInDB.py; included in DatabaseAPI

class is redefined as a method of Database.

AndriesSHP committed:
Delete TestDBcontent.py; replaced by CSV file

Since reading CSV files is implemented, these test data are superfluous.
